apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryRow.cs

```diff
@@ -137,13 +137,16 @@ private static object ConvertSingleValue(object rawValue, TableFieldSchema field
                         return ConvertArray(array, BigNumericConverter);
                     case BigQueryDbType.Geography:
                         return ConvertArray(array, GeographyConverter);
+                    case BigQueryDbType.Json:
```

Comment:
Sorry, I missed this before - another one that can just move to the BigQueryDbType.String block.

Reply:
Thanks for catching this as well. I apparently missed it twice even though I had observed and fixed it in BigQueryParameter.cs in the same PR.